jenkins-plugin/pom.xml

```diff
@@ -33,38 +33,13 @@
   </parent>
 
   <artifactId>pipeline-maven</artifactId>
-  <version>${changelist}</version>
   <packaging>hpi</packaging>
 
   <name>Pipeline Maven Integration Plugin</name>
   <description>
     This plugin provides integration with Pipeline, configures maven environment to use within a pipeline job by calling sh mvn or bat mvn.
     The selected maven installation will be configured and prepended to the path.
   </description>
-  <url>https://github.com/jenkinsci/pipeline-maven-plugin/blob/master/README.adoc</url>
```

Comment:
This broke documentation rendering https://plugins.jenkins.io/pipeline-maven/ due to how the `url` is inherited from the parent pom.

With this change the effective plugin URL became `https://github.com/jenkinsci/pipeline-maven-plugin/blob/master/README.adoc`, since #518 it is `https://github.com/jenkinsci/pipeline-maven-plugin/pipeline-maven`.

Reply:
@daniel-beck Thank you for pointing this ! My bad, I did not think about the Jenkins plugins site

Fixed in https://github.com/jenkinsci/pipeline-maven-plugin/commit/6d9da59877dcdeb9559e95e8195d82e53b433334